src/admin/components/AdminNotice.vue

```diff
@@ -83,7 +87,7 @@ export default {
     methods: {
         fetch() {
             $.ajax( {
-                url: `${dokan_promo.rest.root}${dokan_promo.rest.version}/admin/notices/${this.endpoint}`,
+                url: `${dokan_promo.rest.root}${dokan_promo.rest.version}/admin/notices/${this.endpoint}?scope=${this.scope}`,
```

Comment:
Pass `?scope=${this.scope}` only if the `scope` is not empty.

Reply:
vai, done.